client/my-sites/plans/index.js

```diff
@@ -38,6 +38,14 @@ export default function() {
 		);
 
 		page(
+			'/plans/compare/:product/:domain',
+			retarget,
```

Comment:
I don't understand the purpose of `retarget` here. I actually don't understand the purpose of `retarget` in general.